addons/mail/static/src/components/message/message.js

```diff
@@ -126,24 +126,19 @@ class Message extends Component {
     // Public
     //--------------------------------------------------------------------------
 
+    /**
+     * @returns {string}
+     */
+    get anonymousName() {
+        return (this.message.originThread && this.message.originThread.anonymous_name) || this.env._t("Anonymous");
+    }
```

Comment:
Likewise, this could potentially be a computed field on message.

Reply:
Not sure this would simplify code, having yet another field ... I would prefer to have a placeholder frontend and keep void value backend.

Reply:
Or are you speaking about the JS model ?

Reply:
Yes, a computed field in the JS model :) (already done)

Reply:
Ok, nice